lib/Doctrine/DBAL/Connection.php

```diff
@@ -986,7 +994,7 @@ public function project($query, array $params, Closure $function)
     /**
      * Executes an SQL statement, returning a result set as a Statement object.
      *
-     * @return \Doctrine\DBAL\Driver\Statement
+     * @return DriverStatement|PDOStatement
```

Comment:
The Interface containing `query()` only expects a return type of `Doctrine\DBAL\Driver\Statement`. Should `PDOStatement` be added to the interface too?